src/pybind/cephfs/cephfs.pyx

```diff
@@ -1353,7 +1355,8 @@ cdef class LibCephFS(object):
         self.require_state("mounted")
         path = cstr(path, 'path')
         cdef char* _path = path
-        ret = ceph_rmdir(self.cluster, _path)
+        with nogil:
```

Comment:
oh boy, this has been running since ages I presume and cephfs.pyx has a mix of callouts with and without dropping the GIL.